app/services/booking_responder/booking_request_processor.rb

```diff
@@ -29,7 +29,11 @@ def process_request(message_for_visitor = nil)
 
     delegate :visit, to: :staff_response
     delegate :rejection, to: :visit
+
+    # Remove following ignored rubocop once fix released for conflicting cops
+    # rubocop:disable Style/AccessModifierDeclarations, Layout/AccessModifierIndentation
     private :visit
+    # rubocop:enable Style/AccessModifierDeclarations, Layout/AccessModifierIndentation
```

Comment:
I think that's because `private` keyword should be indented at the class level (which personaly I'm not a big fan of)
so maybe moving this statement just under the other private keyword and indent it at the same level would remove the need for enabling/disabling of the cops

Reply:
@StupidCodeFactory - I've tested moving the ```private :visit``` statement as you suggested, but sadly it still gets stuck in the same loop.

Reply:
well in that case... let's merge this